docs/experiments-ltr-msmarco-passage.md

```diff
@@ -28,12 +28,12 @@ The above scripts convert queries to json objects with `text`, `text_unlemm`, `r
 The first two scripts take ~1 min and the third one is a bit longer (~1.5h).
 
 ```bash
-wget https://www.dropbox.com/s/se5kokw1qqu8yxs/lucene-index-msmarco-passage-ltr.tar.gz -P indexes/ 
-tar -xzvf indexes/lucene-index-msmarco-passage-ltr.tar.gz 
+wget https://www.dropbox.com/s/2h22kt43fcnngxg/index-msmarco-passage-ltr-20210519-e25e33f.tar.gz -P indexes/ 
```

Comment:
with pre-built index, you don't need explicit downloading...

You can do something like:
```
searcher = SimpleSearcher.from_prebuilt_index('msmarco-passage')
```

Directly from your script?

Reply:
hrm, actually we are using index sth like `fe = FeatureExtractor('index-msmarco-passage-ltr-20210519-e25e33f.tar.gz', max(multiprocessing.cpu_count()//2, 1))`
and FeatureExtractor is a class we defined in Anserini... 
we can use pre-built maybe, might need to reorganize java side in a large scale

Reply:
Oh, I see.

How about doing this as a one-liner:

```
python -c "from pyserini.search import SimpleSearcher; SimpleSearcher.from_prebuilt_index('xxxxx')"
```

And then pass the `~/.cache/....` path into Pyserini.

That way, you don't need to worry about the Dropbox path changing again in the future...

Reply:
ah, I see your point. 
yea, that sg.
 next time we change the dropbox link, we only need to update `prebuilt_index.py`

Reply:
> next time we change the dropbox link, we only need to update `prebuilt_index.py`

Correct!